helmfile.d/values/grafana/grafana-user.yaml.gotmpl

```diff
@@ -135,6 +135,12 @@ labels:
 podLabels:
   velero: backup
 
+{{- if .Values.velero.useVolumeSnapshots }}
+persistence:
+  extraPvcLabels:
```

Comment:
This change requires no migration steps?
Since metadata.labels is a mutable field I would assume that it's fine to just apply, but you never know.

Reply:
I will try it out to ensure that no migration script is needed.

Reply:
There was no problem letting helm add the label.
The PVC was not recreated.